src/main/java/duke/storagedata/StorageData.java

```diff
@@ -0,0 +1,214 @@
+package duke.storagedata;
```

Comment:
Writing individually to the file for each command seems quite tedious and hard to maintain. Every single new command requires a new method to be defined.
You might want to implement a `save(TaskList tl)` function which can be used for all kind of commands, and reduce the complexity in this class.